mix.exs

```diff
@@ -1,6 +1,6 @@
 defmodule TaskBunny.Mixfile do
   use Mix.Project
-  @version "0.0.1-dev.2"
+  @version "0.0.1-dev.3"
```

Comment:
It's ok this time but can you bump the version on master after PR has been merged from next time?